projects/plugins/jetpack/modules/calypsoify/class-jetpack-calypsoify.php

```diff
@@ -2,210 +2,145 @@
 /**
  * This is Calypso skin of the wp-admin interface that is conditionally triggered via the ?calypsoify=1 param.
  *
+ * @deprecated $$next-version$$ Use Automattic\Jetpack\Calypsoify instead.
+ *
  * @package automattic/jetpack
  */
 
 use Automattic\Jetpack\Calypsoify\Jetpack_Calypsoify as Calypsoify;
-use Automattic\Jetpack\Status;
 
 /**
  * Class Jetpack_Calypsoify
+ *
+ * @deprecated $$next-version$$
  */
 class Jetpack_Calypsoify {
 
 	/**
 	 * Singleton instance of `Jetpack_Calypsoify`.
 	 *
+	 * @deprecated $$next-version$$
+	 *
 	 * @var object
 	 */
 	public static $instance = false;
 
 	/**
 	 * Is Calypsoify enabled, based on any value of `calypsoify` user meta.
 	 *
+	 * @deprecated $$next-version$$
+	 *
 	 * @var bool
 	 */
 	public $is_calypsoify_enabled = false;
 
-	/**
-	 * Jetpack_Calypsoify constructor.
-	 */
-	private function __construct() {
```

Comment:
Oh, magic methods like constructors are probably an exception to the "you can safely remove private methods". Removing this results in a default public constructor, which we probably don't want to suddenly start allowing. So I'd bring back an empty but still private constructor to preserve that behavior.
```php
	private function __construct() {
	}
```